blockstore/splitstore/splitstore_compact.go

```diff
@@ -184,16 +184,6 @@ func (s *SplitStore) trackTxnRef(c cid.Cid) {
 		return
 	}
 
-	if s.txnProtect != nil {
-		mark, err := s.txnProtect.Has(c)
-		if err != nil {
-			log.Warnf("error checking markset: %s", err)
-			// track it anyways
-		} else if mark {
-			return
-		}
-	}
-
```

Comment:
Why remove this?

Reply:
so that we don't hit the markset and potentially stall an I/O operation.